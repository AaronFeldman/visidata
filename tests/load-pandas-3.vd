sheet	col	row	longname	input	keystrokes	comment	sheet	row	keystrokes	input	longname	undofuncs	col	comment
global		filetype	set-option	pandas			global	filetype		pandas	set-option	[0] 		
			open-file	sample_data/co3.stata	o				o	sample_data/co3.stata	open-file			
co3	s		freq-col		Shift+F	open Frequency Table grouped on current column, with aggregations of other columns	co3		Shift+F		freq-col	[27] [3] <built-in function setattr>; [3] co3_s_freq; _names; [1] co3_s_freq; {0}; [3] <built-in function setattr>; [3] co3_s_freq; _names; [1] co3_s_freq; {0}; [3] <function undoAttrFunc.<locals>._undofunc at 0x7fb4a43f0b80>; [0] ; {0}; [3] <built-in method remove of list object at 0x7fb4a4252280>; [1] <visidata.column.Column object at 0x7fb4a429f370>; {0}; [3] <function undoAttrFunc.<locals>._undofunc at 0x7fb4a43f1480>; [0] ; {0}; [3] <built-in function setattr>; [3] co3_s_freq_cmdlog; _names; [1] co3_s_freq_cmdlog; {0}; [3] <built-in function setattr>; [3] co3_s_freq_cmdlog; _names; [1] co3_s_freq_cmdlog; {0}; [3] <function undoAttrFunc.<locals>._undofunc at 0x7fb4a43cf760>; [0] ; {0}; [3] <built-in function setattr>; [3] co3_cmdlog; _names; [1] co3_cmdlog; {0}; [3] <built-in function setattr>; [3] co3_cmdlog; _names; [1] co3_cmdlog; {0}; [3] <function undoAttrFunc.<locals>._undofunc at 0x7fb4a43f2830>; [0] ; {0}; [3] <built-in function setattr>; [3] co3_s_freq_cmdlog; _names; [1] co3_s_freq_cmdlog; {0}; [3] <built-in function setattr>; [3] co3_s_freq_cmdlog; _names; [1] co3_s_freq_cmdlog; {0}; [3] <function undoAttrFunc.<locals>._undofunc at 0x7fb4a43f2560>; [0] ; {0}; [3] <built-in method remove of list object at 0x7fb4a4359040>; [1] <visidata.column.ItemColumn object at 0x7fb4a42038b0>; {0}; [3] <built-in method remove of list object at 0x7fb4a4359040>; [1] <visidata.column.ItemColumn object at 0x7fb4a42ad150>; {0}; [3] <built-in method remove of list object at 0x7fb4a4e09dc0>; [1] <visidata.column.ItemColumn object at 0x7fb4a42af160>; {0}; [3] <built-in method remove of list object at 0x7fb4a4e09dc0>; [1] <visidata.column.ItemColumn object at 0x7fb4a42b91b0>; {0}; [3] <built-in method remove of list object at 0x7fb4a4e09dc0>; [1] <visidata.column.ItemColumn object at 0x7fb4a42b8df0>; {0}; [3] <built-in method remove of list object at 0x7fb4a4e09dc0>; [1] <visidata.column.ItemColumn object at 0x7fb4a42b9210>; {0}; [3] <built-in method remove of list object at 0x7fb4a4e09dc0>; [1] <visidata.column.ItemColumn object at 0x7fb4a42b9240>; {0}; [3] <built-in method remove of list object at 0x7fb4a4e09dc0>; [1] <visidata.column.ItemColumn object at 0x7fb4a42b9270>; {0}; [3] <built-in method remove of list object at 0x7fb4a4e09dc0>; [1] <visidata.column.ItemColumn object at 0x7fb4a42b92a0>; {0}; [3] <built-in method remove of list object at 0x7fb4a4e09dc0>; [1] <visidata.column.ItemColumn object at 0x7fb4a42b92d0>; {0}; [3] <built-in method remove of list object at 0x7fb4a4252280>; [1] <visidata.column.Column object at 0x7fb4a4203580>; {0}; [3] <built-in method remove of list object at 0x7fb4a4252280>; [1] <visidata.column.Column object at 0x7fb4a4203760>; {0}; [3] <built-in method remove of list object at 0x7fb4a4252280>; [1] <visidata.column.Column object at 0x7fb4a42035e0>; {0}	s	open Frequency Table grouped on current column, with aggregations of other columns
