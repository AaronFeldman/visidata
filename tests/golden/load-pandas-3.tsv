s	count	percent	histogram
1	3	16.67	**************************************************
2	3	16.67	**************************************************
3	3	16.67	**************************************************
4	3	16.67	**************************************************
5	3	16.67	**************************************************
6	3	16.67	**************************************************
